src/main/java/at/hannibal2/skyhanni/config/features/fishing/RareCatchesConfig.kt

```diff
@@ -23,6 +23,14 @@ class RareCatchesConfig {
     @ConfigEditorBoolean
     var alertOtherCatches: Boolean = false
 
+    @Expose
+    @ConfigOption(
+        name = "Announce to Party",
+        desc = "Send a message to your party when you catch a rare sea creature."
+    )
+    @ConfigEditorBoolean
+    var announceRare: Boolean = false
```

Comment:
please include in that variable name the info that this is a announce to party chat, otherwise this variable might get confused with "alertOtherCatches"